### DISCUSSION TITLE
What is the main component of Earth's atmosphere by volume?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#91] [Galaxy Qs] [answered]
What is the main component of Earth's atmosphere by volume ?

**Comment by rezzcode** [comment, marked as answer]
because its made up of 78 percent nitrogen

`nitrogen` definitely is the main component.
its also made up of 21 percent oxygen, 0.9 percent argon, and 0.1 percent other gases
